src/pages/dashboard/Renderer/Renderer/style.less

```
@@ -1,35 +1,44 @@
 .renderer-container {
   height: 100%;
+
   background-color: var(--fc-fill-2);
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)